app/views/citizens/accounts/index.html.erb

```diff
@@ -11,8 +11,48 @@
   We will show your statement from the last 30 days so you can tell us
   more about your financial situation
 </p>
+<div class="govuk-!-padding-bottom-5">
+  <h2 class="govuk-heading-m"><%= t '.your_accounts' %></h2>
+</div>
 
-<p>
-  <strong>Account holders's name</strong>
-  <%= @applicant.first_name %>
-</p>
+<% @applicant_banks.each do |applicant_bank| %>
+
+  <table class="govuk-table">
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="row"><%= t '.acount_holder_name_heading' %></th>
+        <td class="govuk-table__cell"><%= applicant_bank.bank_account_holders.last.full_name %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="row"><%= t '.account_holder_address_heading' %></th>
+        <td class="govuk-table__cell"><%= applicant_bank.bank_account_holders.last.addresses.first.values.join(', ') %></td>
+      </tr>
+    </tbody>
+  </table>
+
+  <table class="govuk-table">
+     <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="col"><%= t '.type_heading' %></th>
+        <th class="govuk-table__header" scope="col"><%= t '.account_number_heading' %></th>
+        <th class="govuk-table__header" scope="col"><%= t '.sort_code_heading' %></th>
+        <th class="govuk-table__header" scope="col"><%= t '.balance_heading' %></th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <% applicant_bank.bank_accounts.each do |bank_account| %>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell"><%= "#{applicant_bank.name} #{bank_account.name}" %></td>
+          <td class="govuk-table__cell"><%= bank_account.account_number %></td>
+          <td class="govuk-table__cell"><%= bank_account.sort_code %></td>
+          <td class="govuk-table__cell"><%= display_value_with_symbol(bank_account.balance, bank_account.currency) %></td>
+        </tr>
+      <% end %>
+    </tbody>
+
+  </table>
+<% end %>
+
+<div class="govuk-!-padding-top-3">
+<%= link_to 'Continue', new_citizens_additional_account_path, class: 'govuk-button', id: 'continue' %>
```

Comment:
**[not a blocker/nit pick]** Probably the lint doesn't detect this, but if we're wrapping something within the div, I would say it should properly indented.